test/auction_test.js

```diff
@@ -57,6 +59,7 @@ contract("SimpleAuction", accounts => {
 
     it("sells an NFT by auction", async () => {
 
+        const mnemonic = process.env.MNEMONIC;
```

Comment:
It should not be necessary to have an environment variable for tests. Just use a static one. Add a grantRole for the address in `scripts/test_setup.js`